src/connectors/index.ts

```diff
@@ -51,15 +49,6 @@ export const injected = new InjectedConnector({
     supportedChainIds: [420],
 })
 
-export const gnosisSafe = new SafeAppConnector()
```

Comment:
Did we lose support for gnosis safe by doing this?

Reply:
we've never supported gnosis safe, the lint command removed this